a.core_concepts.md

```diff
@@ -213,7 +213,7 @@ Alternatively you can also try a more advanced option:
 # Get IP of the nginx pod
 NGINX_IP=$(kubectl get pod nginx -o jsonpath='{.status.podIP}')
 # create a temp busybox pod
-kubectl run busybox --image=busybox --env="NGINX_IP=$NGINX_IP" --rm -it --restart=Never -- wget -O- $NGINX_IP:80
+kubectl run busybox --image=busybox --env="NGINX_IP=$NGINX_IP" --rm -it --restart=Never -- sh -c 'wget -O- $NGINX_IP:80'
```

Comment:
Doesn't the existing example work? I'm in favor of smaller commands, less typing and less to remember for the exam :) Maybe we can add your observation about the variable substitution on a comment.

Reply:
Of course, I'm in favor of smaller commands; the existing example was working "by accident", you can easily check that changing the name of the variable defined inside the POD.

For instance, this won't work:
`kubectl run busybox --image=busybox --env="INSIDE_NGINX_IP=$NGINX_IP" --rm -it --restart=Never -- wget -O- $INSIDE_NGINX_IP:80`

This will work:
`kubectl run busybox --image=busybox --env="INSIDE_NGINX_IP=$NGINX_IP" --rm -it --restart=Never -- sh -c 'wget -O- $INSIDE_NGINX_IP:80'

Reply:
oh, now I get your point, sorry. Definitely agree that your approach is better... Maybe rewrite the sample to differentiate between env inside and outside of the pod? I like your INSIDE_NGINX_IP example!